src/Block/Filter.hs

```diff
@@ -15,6 +16,9 @@ data DatumFilter
   | AddressFilter Text
   deriving stock (Show)
 
+instance Default DatumFilter where
+  def = ConstFilter True
```

Comment:
As I think, in general, the hybrid variant is the best:
```suggestion
instance Default DatumFilter where
  def = defaultDatumFilter
defaultDatumFilter = ConstFilter True
```

But in this case, I agree with @ngua